scripts/publish.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -xe
 
 # Install yarn
-curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --rc
+curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.17.3
```

Comment:
Note for posterity: 1.17 is the version before the integrity hashes were added to the lockfile.  Because we're using our private repo with a self-signed cafile on (some of?) the test servers, but not on the machines that build the lockfile, tests fail those integrity hashes.  By downgrading to 1.17, we skip those failures until we can find a better solution.

Other that the integrity hash issue, nothing keeps us from using 1.19 (or presumably, later versions of yarn).